plugin.py

```diff
@@ -58,7 +80,6 @@ def create_pattern_regexp(self, pattern: str) -> 're.Pattern':
     def on_ready(self, api: ApiWrapperInterface) -> None:
         self._api = api
         self._schema_store.add_listener(self)
-        self._schema_store.load_schemas_async()
```

Comment:
This had to be moved before `on_settings_changed` has triggered later and schema associations notification didn't include user schemas then.